src/main/java/model/blackjackgame/HitAnswer.java

```diff
@@ -0,0 +1,33 @@
+package model.blackjackgame;
+
+public class HitAnswer {
+
+    private static final String YES = "y";
+    private static final String NO = "n";
+    private static final String INVALID_ANSWER = "y 혹은 n 중 하나를 입력해 주세요.";
+
+    private final boolean isHit;
+
+    public HitAnswer(boolean isHit) {
+        this.isHit = isHit;
+    }
+
+    public static HitAnswer of(String isHitAnswer) {
+        boolean isHit = convertYesOrNoToBoolean(isHitAnswer);
+        return new HitAnswer(isHit);
+    }
+
+    private static boolean convertYesOrNoToBoolean(String answer) {
+        if (answer.equals(YES)) {
```

Comment:
```suggestion
        if (YES.equals(answer)) {
```
equals 비교시에 상수가 앞으로 오도록 해주세요!
NPE가 발생할 수 있습니다

Reply:
null safe하게 수정했습니다!😄